docs/spelling_wordlist.txt

```diff
@@ -788,6 +788,7 @@ javascript
 jaydebeapi
 Jdbc
 jdbc
+JdbcHook
```

Comment:
This seems redundant.
Use the ` char when you specify JdbcHook then the spelling check will treat it as code and wont raise issues